### DISCUSSION TITLE
Learn PyTorch for deep learning in a day. Literally. #54

### DISCUSSION THREAD

**Discussion by danmoldashev** [mrdbourke/pytorch-deep-learning#274] [Q&A]
I was completing the code step by step and got stuck on this moment

#plot_predictions(predictions=y_preds) # -> won't work... data not on CPU

#Put data on the CPU and plot it
plot_predictions(predictions=y_preds.cpu())

In my case .cpu() isn't helping to solve the converting from Tensor to numpy

<img width="684" alt="image" src="https://user-images.githubusercontent.com/94741195/214040803-69a3bd24-2bdd-4c00-bf66-b974754ee2e3.png">

**Comment by mrdbourke** [comment]
Hi @danmoldashev,

I've just tried this on my end and the notebook works running end to end.

Have you tried troubleshooting your code step by step?

See the original notebook here: https://github.com/mrdbourke/pytorch-deep-learning/blob/main/01_pytorch_workflow.ipynb
